introduction.md

```diff
@@ -7,3 +7,4 @@ JavaScript
 * Never bring live blades on the tatami
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?